index.html

```diff
@@ -2,12 +2,56 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Hello Microverse!</title>
 </head>
 <body>
     <header>
-        <h1>Server is working!</h1>
+      <nav>
+        <div class="navbar">
+          <a class="logo" href="#">My Logo</a>
+          <a class="menu" href="#">
+            <div class="one"></div>
+            <div class="two"></div>
+            <div class="three"></div>
+          </a>
+       </div>
+      </nav>
+
+      <div class="main">
+        <h1>I'm Mariane Glad to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+        <a class="cta" href="#">let's connect</a>
+        <ul class="socialmedia">
+          <li>
+            <a href="#">
+              <img src="./imgs/twitter.svg" alt="">
+            </a>
```

Comment:
- [ ] Kindly wrap your headline section inside a `<section>` tag instead of a `<div>`.   It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.